corehq/apps/hqcase/management/commands/reindex_sql_forms_in_domain.py

```diff
@@ -0,0 +1,22 @@
+from django.core.management import BaseCommand
+
+from corehq.form_processor.backends.sql.dbaccessors import *
+from corehq.pillows.xform import *
+
+
+def reindex_sql_forms_in_domain(domain):
+    for state, _ in XFormInstanceSQL.STATES:
+        doc_ids = FormAccessorSQL.get_form_ids_in_domain_by_state(domain, state)
+        for form in FormAccessorSQL.get_forms(doc_ids):
```

Comment:
Yeah that would definitely make sense. Also chunked saving below, but that's more of a nice-to-have that would speed things up, whereas this line will easily result in an OOM for even modestly sized domains.